public/class-paystack-forms-public.php

```diff
@@ -62,28 +62,82 @@ public function enqueue_scripts()
 define('KKD_PFF_PAYSTACK_FLATLINE_AMOUNT_PLUS_CHARGE', intval((KKD_PFF_PAYSTACK_LOCAL_CAP-KKD_PFF_PAYSTACK_ADDITIONAL_CHARGE)/KKD_PFF_PAYSTACK_PERCENTAGE));
 define('KKD_PFF_PAYSTACK_FLATLINE_AMOUNT', KKD_PFF_PAYSTACK_FLATLINE_AMOUNT_PLUS_CHARGE - KKD_PFF_PAYSTACK_LOCAL_CAP);
 
-function kkd_pff_paystack_add_paystack_charge($amount)
+class PaystackCharge
```

Comment:
Please namespace this class (prefix with `Kkd_Pff_Paystack_`)

Reply:
Yeaaah. Thanks.